packages/ra-ui-materialui/src/layout/Sidebar.js

```diff
@@ -125,18 +117,11 @@ class Sidebar extends PureComponent {
 Sidebar.propTypes = {
     children: PropTypes.node.isRequired,
     classes: PropTypes.object,
-    closedSize: PropTypes.number,
     open: PropTypes.bool.isRequired,
     setSidebarVisibility: PropTypes.func.isRequired,
-    size: PropTypes.number,
     width: PropTypes.string,
 };
 
-Sidebar.defaultProps = {
-    size: DRAWER_WIDTH,
```

Comment:
then the constants line 12 and 13 can be removed, too - but I think we should use them to set default values for when the theme doesn't contain the sizes.